mkdocs.yml

```diff
@@ -119,6 +119,19 @@ theme:
 
 extra_css: ['overrides.css']
 
+extra:
+  social:
+    - icon: material/web
+      link: https://haveno.exchange
+    - icon: simple/torbrowser # TODO: use actual onion
+      link: http://o64zd74vzijrjchc5ighj2st57mjsvvjty3nhw5pgwj6xolxarlgn3qd.onion
```

Comment:
#33 could make this functionally redundant

Reply:
Thinking of having i2p and onion next to one another

I2p doesnt have the browser feature, and i dont think 33 shows on mobile